news/5600.feature.rst

```diff
@@ -1 +1,3 @@
-Passing ``--use-lock`` to ``pipenv check`` will check again lock file default packages, and specifies ``--categories`` allows specifying which lockfile categories to use.
+Behavior change for ``pipenv check`` now checks the default packages group of the lockfile.
+Specifying ``--categories`` to override which categories to check against.
+Pass ``--use-installed`` to get the prior behavior of checking the packages actually installed into the environment.
```

Comment:
How is categories specified?
Is it a quoted list, e.g.:
```
--categories="foo bar baz" 
```
And of course what is the separtor? space or coma, e.g. `--categories="foo, bar, baz"`
Or do we pass the flag multiple times?
```
--categories foo  --categories bar --categories= baz 
```
Looking at the code, it's `" "` separated, but I think it is appropriate to document this again in the news snippet.

Reply:
@oz123 This PR made it more flexible so you can pass spaces or commas -- https://github.com/pypa/pipenv/pull/5606

Reply:
OK. This is already documented in a past news item.